src/fixtures.js

```diff
@@ -1,25 +1,25 @@
+/* global self */
 'use strict'
 
 const isNode = require('detect-node')
 const path = require('path')
-const Buffer = require('safe-buffer').Buffer
 
-module.exports = function loadFixtures (dirname, file, module) {
+module.exports = function loadFixtures (filePath, module) {
```

Comment:
Could you please add a comment saying that the `filepath` needs to be relative to the module root?

Reply:
done. Also updated the README